src/Network/Network/BYOIP/CustomIpPrefix/UpdateAzureCustomIpPrefixCommand.cs

```diff
@@ -67,7 +67,13 @@ public class UpdateAzureCustomIpPrefixCommand : CustomIpPrefixBaseCmdlet
         public SwitchParameter Commission { get; set; }
 
         [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
-        public SwitchParameter Decomission { get; set; }
+        public SwitchParameter Decommission { get; set; }
```

Comment:
Hey, please add Decomission as the alias of Decommission to avoid breaking change.